test/integration/client.js

```diff
@@ -41,48 +41,48 @@ helpers.getRequest = function(app) {
 helpers.createAndJoinWallet = function(clients, m, n, cb) {
   clients[0].createWallet('wallet name', 'creator', m, n, 'testnet',
     function(err, secret) {
-      if (err) return cb(err);
-      if (n == 1) return cb();
-
-      should.exist(secret);
-      async.each(_.range(n - 1), function(i, cb) {
-        clients[i + 1].joinWallet(secret, 'copayer ' + (i + 1), function(err, result) {
+      should.not.exist(err);
+
+      if (n > 1) {
+        should.exist(secret);
+      }
+
+      async.series([
+
+          function(next) {
+            async.each(_.range(1, n), function(i, cb) {
+              clients[i].joinWallet(secret, 'copayer ' + i, cb);
+            }, next);
+          },
+          function(next) {
+            async.each(_.range(n), function(i, cb) {
+              clients[i].openWallet(cb);
```

Comment:
Ok I tried this:
```clients[i].openWallet(function(err) {
clients[i].isComplete().should.be.true;
return cb();
});```
But then all tests that test against fake copayers joining the wallet would fail...